perma_web/perma/celery_tasks.py

```diff
@@ -1603,3 +1607,22 @@ def warn_expiring_organization_users(warning_days=None):
                 except Exception:
                     logger.exception(f"Error emailing user {affiliation.user_id}")
                 break
+
+
+@shared_task
+def send_user_email_from_bulk_addition(user_email, context, template, host=None, is_new_user=False):
+    """
+    Handles sending emails to users created/updated via the bulk org user addition form
+    """
+    if is_new_user:
+        user = LinkUser.objects.get(raw_email=user_email)
+        activation_route = f'''{host}{reverse('password_reset_confirm', 
+                                              args=[
+                                                urlsafe_base64_encode(force_bytes(user.pk)),
+                                                default_token_generator.make_token(user),
+                                              ])}'''
```

Comment:
Just a suggestion, but I think this will be more legible if you generate the URL outside the f-string:
```suggestion
        path = reverse(
            'password_reset_confirm',
            args=[
                urlsafe_base64_encode(force_bytes(user.pk)),
                default_token_generator.make_token(user),
            ],
        )
        activation_route = f'{host}{path}'
```